### PR TITLE
Feature/core shared dep

### PR DESCRIPTION
**Pull request by AltusBarry**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM praekeltfoundation/python-base:3.6-stretch
 
-RUN apt-get update && apt-get install -y netcat libxml2
+RUN apt-get update && apt-get install -y netcat libxml2 git
```
Comment (cobusc) [inline Dockerfile]: Append `--src /usr/local/src` to the `pip install` line below.

File: project/settings.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+DEFAULT_API_LIMIT = os.environ.get("DEFAULT_API_LIMIT", 50)
+DB_URI = os.environ.get(
+    "DB_URI",
+    "postgres+psycopg2://access_control:password@localhost:5432/access_control"
+)
+API_KEY_HEADER = "X-API-KEY"
+ALLOWED_API_KEYS = set(os.environ["ALLOWED_API_KEYS"].split(","))
+
+# APP Setup
+APP = Flask(__name__)
```
Comment (cobusc) [inline project/settings.py]: Maybe move the app to `app.py` rather than `settings.py`?

File: requirements.txt [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# GIT requirements
+-e git+https://github.com/girleffect/core-shared@feature/db_actions#egg=core-shared
```
Comment (cobusc) [inline requirements.txt]: Is the intention to change this to master before merging?

Comment (AltusBarry) [reply requirements.txt]: Yes, that is the intention.

Comment (AltusBarry) [reply requirements.txt]: It just needs the PR on core-shared merged first

**Comment by cobusc** [comment]
:+1: Thanks, @AltusBarry

**Pull request closed without merging**